inductiva/structures/simulators/fenicsx.py

```diff
@@ -20,21 +20,25 @@ def api_method_name(self) -> str:
 
     def run(
         self,
-        input_dir: types.Path,
```

Comment:
`input_dir` is a mandatory parameter in this method, so don't remove it please! Simply add the new parameters `mesh_filename`, ...

Reply:
The challenge lies in situations where you already possess the meshes dataset, bc dataset, and material dataset, but you want to avoid duplicating each file in multiple directories. Otherwise, the same data would be copied across different directories, especially if the same mesh is being used. What do you believe would be the most effective approach to address this issue? One possible solution could be to selectively copy only the required files from the mesh, bc, and material datasets for each simulation, which might indeed be the best way to handle this situation.